cloudslang-entities/src/test/java/io/cloudslang/lang/entities/DeserializeTest.java

```diff
@@ -14,9 +16,16 @@
 
 public class DeserializeTest {
 
-    private ObjectMapper mapper = new ObjectMapper();
+    private ObjectMapper mapper;
     Gson gson = new Gson();
 
+    public DeserializeTest() {
```

Comment:
Should not use constructor for Junit.
I think it is better to use a method with the `@BeforeClass` annotation